lldb/test/API/tools/lldb-dap/console/TestDAP_console.py

```diff
@@ -4,10 +4,30 @@
 
 import dap_server
 import lldbdap_testcase
+import psutil
+from collections import deque
 from lldbsuite.test import lldbutil
 from lldbsuite.test.decorators import *
 from lldbsuite.test.lldbtest import *
 
+def get_subprocess_pid(process_name):
+    queue = deque([psutil.Process(os.getpid())])
+    while queue:
+        process = queue.popleft()
+        if process.name() == process_name:
+            return process.pid
+        queue.extend(process.children())
+
+    print(f"No subprocess with name {process_name} found", flush=True, file=sys.stderr)
+    return None
+
+def killProcess(pid, process_name):
+    process = psutil.Process(pid)
+    process.terminate()
+    try:
+        process.wait(timeout=5)
+    except psutil.TimeoutExpired:
+        self.assertTrue(False, process_name + " process should have exited by now")
```

Comment:
you don't need an assert here if you want the test to fail, you can just let the exception bubble up